### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by j-tulforov**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ import { getTotalRewards } from '@/statistics/getTotalRewards';
 import { getStSolAccountsForUser } from '@/stake/getStSolAccountsForUser';
 import { ErrorWrapper } from '@/utils/errorWrapper';
 
-export { default as LidoStakeBanner } from './banner';
+// export { default as LidoStakeBanner } from './banner';
```
Comment (Tataklim) [inline src/index.ts:35]: why did yo comment it?

File: src/types.ts [hunk 2]
```diff
@@ -188,7 +188,7 @@ export type ValidatorV2 = {
   // The result is stored on-chain to optimize compute budget
   effective_stake_balance: BN;
   // Controls if a validator is allowed to have new stake deposits
-  active: boolean;
+  active: boolean | 0 | 1;
```
Comment (Tataklim) [inline src/types.ts:191]: Why is this necessary?

File: tests/data/snapshot.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+export const validators: TestValidator[] = [
+  {
+    vote_account_address: new PublicKey('69rUCD1qk6hJ3AqEkCi236QaceHTBSv1isiCEW6EZnVa'),
+    stake_seeds: { begin: new BN('3'), end: new BN('4') },
+    unstake_seeds: { begin: new BN('0'), end: new BN('0') },
+    stake_accounts_balance: new BN('15881359081'),
+    unstake_accounts_balance: new BN('0'),
+    effective_stake_balance: new BN('15881359081'),
+    active: 1,
```
Comment (Tataklim) [inline tests/data/snapshot.ts:13]: Why 1, not "true"? What is the difference?

File: tests/data/snapshot.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import BN from 'bn.js';
+import { PublicKey } from '@solana/web3.js';
+import { TestValidator } from '../updateSnapshot';
+
+export const validators: TestValidator[] = [
```
Comment (Tataklim) [inline tests/data/snapshot.ts:5]: Are these real validators?

File: tests/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { Connection } from '@solana/web3.js';
+
+const CUSTOM_ENDPOINT = 'https://pyth-testnet-rpc-1.solana.p2p.org/yIwMoknPihQvrhSyxafcHvsAqkOE7KKrBUpplM5Xf';
```
Comment (Tataklim) [inline tests/helpers.ts:3]: this rpc is for testnet - so we can't run tests for mainnet, right?

File: tests/solidosdk.test.ts [hunk 6]
```diff
@@ -0,0 +1,73 @@
+import { SolidoSDK, TX_STAGE } from '@/index';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { ERROR_CODE, ERROR_CODE_DESC, ERROR_MESSAGE } from '@/constants';
+
+describe('SolidoSDK', () => {
+  const walletWithStSolTokenAccount = new PublicKey('2Vn1xSUTo292A3knejUeifjt2A3aGNqyn9Svy8Kx8i4J');
```
Comment (Tataklim) [inline tests/solidosdk.test.ts:6]: As I can see u have this address in consts

File: tests/solidosdk.test.ts [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+  const cluster = 'testnet';
```
Comment (Tataklim) [inline tests/solidosdk.test.ts:7]: I think we can put the cluster in constants. And depending on the selected cluster, substitute the necessary values - for the wallet address, rpc address, etc.

File: tests/validatorsDumps.ts [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  if (validatorListResp === null) {
+    console.log('getAccountInfo of validatorList returned null, try again');
+    return;
+  }
+
+  const deserializedValidators = deserializeUnchecked(
+    validatorsSchema,
+    AccountList,
+    validatorListResp.data,
+  ) as any as ValidatorsList;
```
Comment (Tataklim) [inline tests/validatorsDumps.ts:41]: Why just  `) as ValidatorsList;` is not enough? As I can see no errors appear after this change

File: tests/validatorsDumps.ts [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+import BN from 'bn.js';
+
+import fs from 'fs';
+import path from 'path';
+
+import { AccountList, validatorsSchema } from '@/unstake';
+import { VALIDATOR_LIST } from '@/constants';
+import { ValidatorsList } from '@/types';
+
+const connection = new Connection(
```
Comment (Tataklim) [inline tests/validatorsDumps.ts:12]: Maybe we can put this endpoint to consts?

File: tests/statistics/getMarketCap.test.ts [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+  let sdk, connection;
+
+  beforeAll(() => {
+    connection = getConnection();
+    sdk = new SolidoSDK(cluster, connection);
+
+    global.fetch = jest.fn();
+  });
+
+  test('market cap returned valued', async () => {
```
Comment (Tataklim) [inline tests/statistics/getMarketCap.test.ts:16]: How about test got getMarketCap throwing error?

File: tests/data/snapshot.ts (hunk 4 above)
Comment (j-tulforov) [reply tests/data/snapshot.ts:5]: Real validators on testnet
